frontends/ol-components/src/components/LearningResourceExpanded/LearningResourceExpandedV2.tsx

```diff
@@ -410,7 +410,11 @@ const ImageSection: React.FC<{
 const getCallToActionUrl = (resource: LearningResource) => {
   switch (resource.resource_type) {
     case ResourceTypeEnum.PodcastEpisode:
-      return resource.podcast_episode?.episode_link
+      return (
+        resource.podcast_episode?.episode_link ||
+        resource.podcast_episode?.audio_url ||
+        resource.url
+      )
     default:
       return resource.url
```

Comment:
Now that podcasts use `"url": episode_link or audio_url, the switch statement seems unnecessary to me.

In fact, could we just get rid of `getCallToActionUrl` and do `href={resource.url}` in the component?